src/docs/santiment-queries/metric-tables/index.md

```diff
@@ -0,0 +1,374 @@
+---
+title: Metrics Datasets
+author: Santiment Team
+date: 2024-04-01
+description: Learn about Santiment metrics datasets.
+---
+
+## Overview
+
+This document introduces Santiment's precomputed metrics datasets.
+
+You can filter all metric tables as:
+
+```sql
+SHOW TABLES LIKE '%metric%'
+```
+
+Test in [Queries](https://queries.santiment.net/query/list-of-tables-390)
```

Comment:
The example shows tables with `like '%price%'`

Reply:
Fixed